src/holon/services/query_and_convert.py

```diff
@@ -208,10 +209,14 @@ def set_anylogic(self, c: AnyLogicConversion):
                     f"Couldn't find the specified key '{c.anylogic_key}' in any of the AnyLogic results (resort to convert with 1)"
                 )
         except:
-            pprint(
-                f"Found the key '{c.anylogic_key}' but the result does not parse to a float (resort to convert with 1)"
-            )
-            value = 1
+            # try to get the values from the dict, to check if dict
+            try:
+                value = list(value.values())[:8760]
```

Comment:
Waar komt magic number 8760 vandaan?

Reply:
Is om ervoor te zorgen dat het 8760 uur is, soms maakt AnyLogic er 8761 van, dat mag niet.